packages/_common/src/frame-processor.ts

```diff
@@ -163,6 +164,31 @@ export class FrameProcessor implements FrameProcessorInterface {
     if (!this.active) {
       return {}
     }
+
+    if (this.paused) {
+
+      const audioBuffer = this.audioBuffer
+
+      const speechFrameCount = audioBuffer.reduce((acc, item) => {
+        return acc + +item.isSpeech
+      }, 0)
+
+      this.active = false
+      this.paused = false
+      this.reset()
+
+      const isSpeech = 0
+      const notSpeech = 1 - isSpeech
+      const probs = { notSpeech, isSpeech }
+
+      if (speechFrameCount >= this.options.minSpeechFrames) {
+        const audio = concatArrays(audioBuffer.map((item) => item.frame))
+        return { probs, msg: Message.SpeechEnd, audio }
+      } else {
+        return { probs, msg: Message.VADMisfire }
```

Comment:
We should not return `VADMisfire` if `this.speaking` is false, which means that `SpeechStarted` is not triggered yet.

Reply:
@antoine-lizee Could you handle this? Or I will make another PR instead.